UKHO.ADM/Functions/Update-AD.ps1

```diff
@@ -4,31 +4,31 @@ function Update-AD {
         $cd,
         [bool]$ApplyChanges = $false
     )
-    begin {}
+    begin {
+    }
     process {
-        Write-Colour -LinesBefore 1 "Actions are indicated with the following symbols:" -Color White
-        Write-Colour -StartTab 1 "+", " adding or creating" -Color Green, White
-        Write-Colour -StartTab 1 "-", " removing or deleting" -Color Red, White
-        Write-Colour -StartTab 1 "~", " modification or change" -Color Yellow, White
-        Write-Colour -StartTab 1 "x", " errors have occurred"  -Color Magenta, White
-
         $out = Generate-ConfigurationObject -ConfigData $cd 
 
-        Write-Colour -LinesBefore 1 "UKHO.ADM will perform the following changes:" -Color Blue
+        $ADChanges = Get-ADChanges -ADDomains $out
```

Comment:
This is now in a different place, above where it was before. Was this on purpose?

It will now output 

- Output of all the changes
- Description of the symbols
- A line saying - "UKHO.ADM will perform the following changes:"
- Change summary

It might be better this way around(save having to scroll to the top maybe to see the description) but dunno. Just checking if on purpose(not mentioned in PR description)

Reply:
All of the Write-Colours lines are now after the Get-ADChanges call as this instanciates the SystemColour properties.

Reply:
alternative might be to instantiate ADChanges before passing into Get-ADChanges

Reply:
Moved the Write-Colour call for the legend types into the creation point of the ADChanges object

Moved all the other "process" points to other areas in the code.